go/bind/keybase.go

```diff
@@ -81,6 +82,9 @@ func Init(homeDir string, logFile string, runModeStr string, accessGroupOverride
 		fmt.Printf("Go: Using log: %s\n", logFile)
 	}
 
+	// Set to one OS thread on mobile so we don't have too much contention with JS thread
+	runtime.GOMAXPROCS(1)
```

Comment:
maybe get the return value here and print it, just for information as to what it was before.

Reply:
Good idea.